extensions/azurehybridtoolkit/notebooks/hybridbook/content/hadr/configure-failover.ipynb

```diff
@@ -37,25 +37,23 @@
                 "- One or more domain joined VMs in Azure running SQL Server 2016 (or later) in the _same_ availability set or _different_ availability zones that have been registered with the SQL IaaS Agent extension\n",
                 "- Domain joined VMs must have either a \"Enterprise\" or \"Developer\" image SKU to create a cluster in this notebook\n",
                 "\n",
-                "Tutorial on how to set up a domain with VMs can be found [here.](https://docs.microsoft.com/en-us/azure/azure-sql/virtual-machines/windows/availability-group-manually-configure-prerequisites-tutorial)\n",
+                "Tutorial on how to set up a domain with VMs can be found [here](https://docs.microsoft.com/en-us/azure/azure-sql/virtual-machines/windows/availability-group-manually-configure-prerequisites-tutorial) (only follow the steps up to setting the SQL Server Service Accounts)\n",
```

Comment:
Avoid referring to a hyperlink with "here" in the <a> tag. Instead, create a sentence like: "Refer to the Tutorial on How to Setup a Domain with VMs in Azure" and placing the tag around the name of the tutorial.